.restyled.yaml

```diff
@@ -2,5 +2,7 @@ restylers:
   - name: prettier
     arguments: ['--write']
     include:
+      - './*.js'
+      - 'src/**/*.js'
```

Comment:
interesting - restyled doesn't seem happy with the JS even though `eslint` is.

Reply:
seems like [`eslint` isn't supported (yet)](https://docs.restyled.io/available-restylers)

Reply:
we could

- drop `eslint` and just use `prettier` for everything, or
- leave things as they are & merge now

I don't mind

Reply:
What if we run `eslint` and `prettier` in that order as part of the `package.json` predefined commands?

Reply:
Or, maybe, https://github.com/prettier/eslint-config-prettier?

Reply:
https://github.com/iterative/setup-dvc/blob/a943f4ff3a84b06555099c5f6c7e2b858ec88535/package.json#L27-L32
Would running `prettier` after `eslint` solve this conundrum?

Reply:
hmm... https://github.com/iterative/iterative.ai/blob/3565fb35836497419e8c7adde256bedc829e8be8/package.json#L84-L85

while https://github.com/iterative/cml-website/blob/master/package.json seems to have unused `eslint` config. Need to find a js packaging expert :/

Reply:
> _Need to find a js packaging expert_

Definitely not me: snakes, gophers and crustanceans don't have any coffee as part of their diet. 😄